sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md

```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry Exporter for Azure Monitor Client for Python
+
```

Comment:
nit: why not include in the same readme?

Reply:
Taking a look at the command for autorest: `autorest ./swagger/README_QUICKPULSE.md --python --v3`, it takes a single README file and i dont believe you can define multiple `yamls` in the same README.

Reply:
Oh didn't realize it was functional too. Alright